### PR TITLE
Truncate outlink

### PR DESCRIPTION
**Pull request by Aumarh**

### CODE REVIEW

File: frontend/pages/article/[slug].tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ const Article: NextPage = () => {
         data && <ModalShare
           isVisible={isCopyModalVisible}
           onCancel={() => setCopyModalVisible(false)}
-          textLink={data?.url}
+          textLink={data?.url.slice(0, 30) + "..."}
```
Comment (hynnot) [inline frontend/pages/article/[slug].tsx:41]: I think that this is not exactly what Paula was commenting on, I think the idea is to keep only the base url as shown in her example.

_Instead of containing the whole URL, the link text should be truncated to [subdomain].[domain].[top level domain] 
E.g. from “https://www.nytimes.com/2023/03/08/opinion/noam-chomsky-chatgpt-ai.html” to “[nytimes.com](http://nytimes.com/)”_

**Review by hynnot — CHANGES_REQUESTED** [review]

File: frontend/pages/article/[slug].tsx (hunk 1 above)
Comment (Aumarh) [reply frontend/pages/article/[slug].tsx:41]: ok, let me implement that.

**Review by dimakrsna — CHANGES_REQUESTED** [review]
If I understand the task correctly, the only thing you need to change is the visual part (to display the top of the domain), but the whole link should be copied, not the cut off part.

Now the link is not displayed correctly and its cropped part is copied.

This may require some changes to the component  `<ModalShare />`

**Comment by Aumarh** [comment]
> If I understand the task correctly, the only thing you need to change is the visual part (to display the top of the domain), but the whole link should be copied, not the cut off part.
> 
> Now the link is not displayed correctly and its cropped part is copied.
> 
> This may require some changes to the component `<ModalShare />`

@dimakrsna, please could you take a look at the edits.

**Comment by dimakrsna** [comment]
@Aumarh The current implementation does not match the task description.

**Review by dimakrsna — CHANGES_REQUESTED** [review]
The current implementation does not match the task description.

**Pull request closed without merging**